tests/helpers.py

```diff
@@ -1572,3 +1572,17 @@ def wait_for_ct_pod_recovery():
         else:
             return False
     return True
+
+
+def label_worker_node(node_list, label):
```

Comment:
I'm sure you'll also need a function to remove the label after the test. can you also add that functionality?

Reply:
ack

Reply:
added code for removing label, resolving this comment